fma_project_spec.ipynb

```diff
@@ -0,0 +1,224 @@
+    "\n",
+    "> Using a dataset of song properties, apply machine learning methods in Python to classify tracks by genre.\n",
+    "\n",
+    "---\n",
+    "\n",
+    "#### Long description ####\n",
+    "\n",
+    "A longer description of the project, around four sentences in length. \n",
+    "This will be read by the students on the DataCamp platform **before** deciding to start the project.\n",
+    "\n",
+    "It should mention some of the major prerequisites for completing the project \n",
+    "(for example \"familiarity with `pandas` DataFrames\" or \"know how to use the Naive Bayes method from `scikit learn`\")\n",
+    "\n",
+    "> Using a subset of the dataset comprised of two genres (Hip-Hop and Rock), we will train a classifier to distinguish between the two genres based only on track infromation derived from Echonest (now part of Spotify) data for each track. We will first make use of `pandas` and `seaborn` packages in Python for subsetting the data, aggregating information, and creating plots when exploring the data for obvious trends or factors we should be aware of when doing machine learning. Next, we will use of the `scikit-learn` package to predict whether we can correctly classify a song's genre based on features such as danceability, energy, acousticness, tempo etc. We will go over implementations of common algorithms such as logistic regression, support vector machines, k-nearest neighbours clustering, decision trees and so forth.\n",
+    "\n",
+    "\n",
+    "#### Datasets used ####\n",
+    "\n",
+    "Short description (and ideally links) to the datasets used in the project. This will be read my me (David) only.\n",
+    "\n",
+    "> An open dataset for music analysis - https://github.com/mdeff/fma. The tracks represented in the dataset are all from the Free Music Archive, a large library of free audio downloads. \n",
+    "\n",
+    "TODO: add links to datasets - these can be links to GitHub uploads\n",
+    "\n",
+    "#### Assumed student background ####\n",
+    "\n",
+    "What background knowledge you assume the student doing this project will have. The more specific the better. This will be read my me (David) only. Please list things like modules, tools, functions, methods, and statistical concepts and jargon.\n",
+    "\n",
+    "Not so useful: \"The student has a basic familiarity with `pandas`.\"\n",
+    "\n",
+    "More useful: \"The student knows how to read in a csv file using `pandas` and how to compute grouped summary statistics using `groupby()`.\"\n",
+    "\n",
+    "> The student knows how to read in csv and json files using `pandas`, and examine correlations across the columns of a `pandas` DataFrame. The student understands basic plotting (scatter plots, pair plots) with `seaborn`. The student is also familiar with implementing both principal component analysis and supervised learning methods (k-fold crossvalidation) in `scikit-learn`."
```

Comment:
Suggestion for the last sentence:

The student is also familiar with implementing principal component analysis, supervised learning methods (decision trees and logistic regression), and model accuracy scoring using k-fold crossvalidation and  the classification report in scikit-learn.

@linamnt Do we want to talk about precision and recall and plot the curves in the final version?